CONTRIBUTING.md

```diff
@@ -0,0 +1,102 @@
+# Contributing
+
+Open a pull request to contribute a change.
+
+## Commit message format
+
+The project is using [semantic-release](https://semantic-release.gitbook.io/semantic-release/)
+and commit lint which enforces formatting rules on commit messages.
+
+This is a modified version from [Angular](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#-commit-message-format)
+project.
+
+Each commit message consists of a **header**, a **body**, and a **footer**.
+
+
+```
+<header>
+<BLANK LINE>
+<body>
+<BLANK LINE>
+<footer>
+```
+
+The `header` is mandatory and must conform to the [Commit Message Header](#commit-header) format.
+
+The `body` is mandatory for all commits except for those of type "docs".
+When the body is present it must be at least 20 characters long and must conform to the [Commit Message Body](#commit-body) format.
+
+The `footer` is optional. The [Commit Message Footer](#commit-footer) format describes what the footer is used for and the structure it must have.
+
+
+### <a name="commit-header"></a>Commit Message Header
+
+```
+<type>(<scope>): <short summary>
+  │       │             │
+  │       │             └─⫸ Summary in present tense. Not capitalized. No period at the end.
+  │       │
+  │       └─⫸ Commit Scope: config|selinux|rpm|daemon|hostinfo|notify|packaging|changelog|
+  |                          makefile
+  │
+  └─⫸ Commit Type: build|ci|docs|feat|fix|perf|refactor|test
+```
+
+The `<type>` and `<summary>` fields are mandatory, the `(<scope>)` field is optional.
+
+
+#### Type
+
+Must be one of the following:
+
+* **build**: Changes that affect the build system or external dependencies (example scopes: makefile)
+* **ci**: Changes to our CI configuration files and scripts (examples: GitHub Actions)
+* **docs**: Documentation only changes
+* **feat**: A new feature
+* **fix**: A bug fix
+* **perf**: A code change that improves performance
+* **refactor**: A code change that neither fixes a bug nor adds a feature
+* **test**: Adding missing tests or correcting existing tests
+
+#### Check commit message locally:
+
+```
+$ npm install
+$ npx commitlint --from HEAD~1
+```
+
+## Release
+
+Is using semantic release with semantic versioning.
+
+Semantic-release is configured in `package.json`.
+
+It:
+- determines the current version based on git tags
+- determines the next version based on commit messages
+- bumps version in version/version.go
+- generates CHANGELOG.md update
+- commits the changes
+- creates a git tag
```

Comment:
Does it sign the git tag?

Reply:
Most likely not -  I don't see it and this code https://github.com/semantic-release/semantic-release/blob/master/lib/git.js#L226 doesn't look to use it.

Reply:
Bummer :/ I have been preferring workflows that require a human to create and sign the tag (if there is no service to do so in a secure way) and then pushing the tag to kick off the release process.